lib/console.js

```diff
@@ -249,6 +252,74 @@ Console.prototype.groupEnd = function groupEnd() {
     this[kGroupIndent].slice(0, this[kGroupIndent].length - 2);
 };
 
+const untablableTypes = [
+  'symbol', 'undefined', 'boolean', 'number', 'string', 'function',
+];
+
+const untablableValues = [
+  undefined, null, NaN, true, false, global, process,
+];
+
+const untablable = (value) =>
+  untablableValues.includes(value) ||
+  untablableTypes.includes(typeof value);
+
+function getTablableKeysOf(O) {
+  if (isSet(O))
+    return Array.from({ length: O.size }, (e, i) => i);
```

Comment:
The spec says it should log the properties. We could show those and add a special handling for them but e.g. Chrome does not show them.

The same applies to the entries below.